views/js/controller/Reporting/index.js

```diff
@@ -114,6 +131,53 @@ define([
                 }
                 return '';
             }
+
+            var datatableTools = [
+                {
+                    id: 'download',
+                    icon: 'download',
+                    title: __('Download the selected reports to a CSV file'),
+                    label: __('Download CSV'),
+                    action: function() {
+                        notYet();
+                    },
+                }, {
+                    id : 'printRubric',
+                    title : __('Print rubric block with item tagged with tao-print tag'),
+                    icon : 'print',
+                    label : __('Print Score Report'),
+                    action : printRubric
```

Comment:
This tool button should only appear when lines are selected.
Please use the property `massAction: true`.